playbooks/test.yml

```diff
@@ -21,18 +21,97 @@
 
   tasks:
   - block:
+    # - name: copy java program to zos
+    #   zos_copy:
+    #     src: /Users/adn/workspaces/nazare/nazare-demo-imsapp/nazare-ims-jmp/null/nazare-ims-jmp-1.0.0-SNAPSHOT.jar
+    #     dest: /u/ibmuser/ims/nazare-ims-jmp-1.0.0-SNAPSHOT.jar
+    #     is_binary: true
+    #     force: true
+
+    # - name: copy ims mpr options to zos
+    #   zos_copy:
+    #     src: /Users/adn/workspaces/nazare/nazare-demo-imsapp/UnixFiles/dfsjvmpr.props
+    #     dest: /u/ibmuser/ims/dfsjvmpr.props
+    #     force: true
```

Comment:
Do we need to keep these lines commented on the code? Looks like this is still some work in progress to add a java program.
I think removing it or changing the src directory to a generic location would work.

Reply:
Just noticing this is a test playbook, do we even want to keep track of it here on GH? 
This could be just for local work.